### ISSUE TITLE
Implement Pre-Training🛝

### ISSUE THREAD

**Issue by KarelZe** [KarelZe/thesis#271]
- [x] Add notebook
- [x] Add to `TransformerClassifier.py` / `objective.py`
- [x] Add sampling on unlabelled data to `train_model.py`.
- [x] Add disable weight decay for norm, bias, and tokenizer in AdamW
